lit-mar/config.json

```diff
@@ -0,0 +1,3 @@
+{
+  "url_mar": "https://s3.us-east-2.amazonaws.com/baiachen-amazon-ai-work-data/img_classification.mar"
```

Comment:
consider spelling it out for readability - `model_archive_url`

Reply:
Modified accordingly